layouts/_partials/language-switch.html

```diff
@@ -10,12 +14,13 @@
     <button
       title="{{ $changeLanguage }}"
       data-state="closed"
-      class="hextra-language-switcher hx:cursor-pointer hx:h-7 hx:rounded-md hx:px-2 hx:text-left hx:text-xs hx:font-medium hx:text-gray-600 hx:transition-colors hx:dark:text-gray-400 hx:hover:bg-gray-100 hx:hover:text-gray-900 hx:dark:hover:bg-primary-100/5 hx:dark:hover:text-gray-50 hx:grow"
+      data-location="{{ $location }}"
+      class="hextra-language-switcher hx:cursor-pointer hx:h-7 hx:rounded-md hx:text-left hx:font-medium {{ $class }} hx:grow"
       type="button"
       aria-label="{{ $changeLanguage }}"
     >
       <div class="hx:flex hx:items-center hx:gap-2 hx:capitalize">
-        {{- partial "utils/icon" (dict "name" "globe-alt" "attributes" "height=12") -}}
+        {{- partial "utils/icon" (dict "name" "globe-alt" "attributes" (printf "height=%d" $iconHeight)) -}}
```

Comment:
```suggestion
        {{- partial "utils/icon" (dict "name" "translate" "attributes" (printf "height=%d" $iconHeight)) -}}
```

"translate" could be a more suitable icon here

Reply:
To avoid a kind of regression, I kept `globe-alt` as a default for the partial (use inside the footer), and used `translate` as a default for the navbar.
I also added an option to the navbar parameter to change the icon.